src/libraries/Common/src/Interop/Android/System.Security.Cryptography.Native.Android/Interop.EVP.Cipher.cs

```diff
@@ -0,0 +1,235 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class Crypto
```

Comment:
If possible, rename to AndroidCrypto.  (If there's still too much code sharing, then I guess it has to wait)

Reply:
Same issue with removing the Evp prefix on the managed names: We're still sharing the AesImplementation class, so I don't want to split that in this PR just for renaming purposes.